lib/gym/xcodebuild_fixes/package_application_fix.rb

```diff
@@ -42,6 +42,21 @@ def patch_package_application
 
         return @patched_package_application_path # Return path to the patched PackageApplication
       end
+
+      # Wrap xcodebuild to work-around ipatool dependecy to system ruby
+      def wrap_xcodebuild
+        require 'fileutils'
+
+        @wrapped_xcodebuild_path = File.join("/tmp", "xcbuild-safe.sh")
+
+        unless File.exist?(@wrapped_xcodebuild_path)
```

Comment:
I originally meant it like that because of:
* cleaner command line
* easy to access the script to modify it and adapt it while developing
* similar to the way we install other wrapper scripts.

But I can see that this can cause problems with upgrades as is (it your newer version of gym contains an improved wrapper, it won't be copied over).

So I'll make it run in place instead. I'll fix it later today.